internal/cmd/kafka/confluent_kafka_configs.go

```diff
@@ -71,6 +71,14 @@ func getConsumerConfigMap(group string, kafka *configv1.KafkaClusterConfig, clie
 		return nil, err
 	}
 	log.CliLogger.Debugf("Created consumer group: %s", group)
+	if commitGroupOffset {
+		if err := configMap.SetKey("enable.auto.commit", true); err != nil {
+			return nil, err
+		}
+		if err := configMap.SetKey("auto.commit.interval.ms", 100); err != nil {
```

Comment:
Ah, I think this is what I was doing wrong earlier. The default is 5s...

Reply:
What are your thoughts on manually committing? What if the client closes before the 100ms timer goes off?

Reply:
(I think that's what `kcat` does)

Reply:
how to manually commit? hitting a key or running a command?

Reply:
We can capture the interrupt from the ctrl-c key, or maybe even use a `defer` in case the program errors?

Reply:
`defer` sounds safer. changing that!

Reply:
or we can call `commit` every time a message is being consumed.

Reply:
added a defer in `runConsumer`

Reply:
:+1: Make sure it also runs when Ctrl-C is pressed!

Reply:
(Also, do we still need to turn on auto-commit if we're manually committing?)

Reply:
let's keep auto-commit on but use a longer interval so it doesn't impact the performance.

Reply:
What's the reason for keeping auto-commit on?

Reply:
on the second thought. auto-commit is for convenience. the way we're doing now is more precise control.
turning it off.